packages/components/button/README.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: 'Button'
+type: 'component'
+status: 'stable'
+slug: /components/button/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/components/button'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-button--button'
+typescript: ./src/Button.tsx
+---
+
+import { Props } from '@contentful/f36-docs-utils';
```

Comment:
What's the deal with this line ?

Reply:
This helps with displaying components props on the documentation website, see https://github.com/contentful/forma-36/pull/888#issue-598196024